lib/usdUfe/ufe/UsdSceneItemOps.h

```diff
@@ -51,6 +49,7 @@ class MAYAUSD_CORE_PUBLIC UsdSceneItemOps : public Ufe::SceneItemOps
         else
             return PXR_NS::UsdPrim();
     }
+    UsdSceneItem::Ptr usdSceneItem() const { return _item; }
```

Comment:
Added new method to get the stored UsdSceneItem - used by the derived version in MayaUsd.